docs/config_search.md

```diff
@@ -16,10 +16,14 @@ limitations under the License.
 
 # Model Config Search
 
-Model Analyzer's `profile` subcommand supports **automatic** and **manual**
+Model Analyzer's `profile` subcommand supports [Brute](config_search.md#brute-search-mode) and [Quick](config_search.md#quick-search-mode) search modes.
+
+## Brute Search Mode
+
+Model Analyzer's brute search mode supposts **automatic** and **manual**
```

Comment:
This should explain which conditions lead to automatic vs manual, and the automatic and manual words should be links.

Also, typo "supposts"

Reply:
Fixed